script.module.tmdbsimple/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.tmdbsimple"
+       name="tmdbsimple"
+       version="2.2.0+matrix.1"
+       provider-name="Celia Oakley, rmrector">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+    <import addon="script.module.requests" version="2.15.1"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">A wrapper for The Movie Database API v3</summary>
+    <description lang="en_GB">Packed for Kodi from https://github.com/celiao/tmdbsimple</description>
+    <platform>all</platform>
+    <license>GPL-3.0</license>
```

Comment:
GPL-3.0-only or GPL-3.0-or-later (depending on the license you bundle)

Reply:
Do you want to adjust the license identifier as part of this PR or do you want to adjust it in a future update?

Reply:
Later works for me.